src/overlays/actors/ovl_Bg_Po_Syokudai/z_bg_po_syokudai.c

```diff
@@ -1,15 +1,48 @@
+/*
+ * File: z_bg_po_syokudai.c
+ * Overlay: ovl_Bg_Po_Syokudai
+ * Description: Golden Torch Stand (Poe Sisters)
+ */
+
 #include "z_bg_po_syokudai.h"
 
 #define FLAGS 0x00000000
 
 #define THIS ((BgPoSyokudai*)thisx)
 
+#define POE_SISTER_PURPLE 0 // Meg
+#define POE_SISTER_RED 1    // Joelle
+#define POE_SISTER_BLUE 2   // Beth
+#define POE_SISTER_GREEN 3  // Amy
+
+#define IS_POE_SISTER_BEATEN(color) Flags_GetSwitch(globalCtx, 0x1C + POE_SISTER_##color)
+#define IS_THIS_POE_SISTER_BEATEN() Flags_GetSwitch(globalCtx, this->actor.params)
```

Comment:
Personally not a fan of these macros, at least right now. I think this kind of stuff should be introduced once the other poe sister actors are decompiled (and merged) and we want to document them in a coherent manner.